src/payment-upon-iot/grammar/template.tem

```diff
@@ -0,0 +1,3 @@
+Upon long button press, [{buyer}] shall pay to [{seller}] [{amountPerUnit}] for each short button press. A maximum of [{paymentCount}] payments may be made before this contract is COMPLETED.
```

Comment:
I would be interested in getting a legal review on the text to know whether such a text could be legally enforceable.